client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +201,226 @@ def process(self, instance):
+                " out as an OSubD and crease info will be stored in the "
+                "Alembic  file. Otherwise, creases info won't be preserved in "
+                "Alembic file unless a custom Boolean attribute "
+                "SubDivisionMesh has been added to mesh node and its value is "
+                "true."
+            ),
+            "dontSkipUnwrittenFrames": (
+                "When evaluating multiple translate jobs, this decides whether"
+                " to evaluate frames between jobs when there is a gap in their"
+                " frame ranges."
+            ),
+            "eulerFilter": "Apply Euler filter while sampling rotations.",
+            "noNormals": (
+                "Present normal data for Alembic poly meshes will not be "
+                "written."
+            ),
+            "preRoll": "This frame range will not be sampled.",
+            "renderableOnly": "Only export renderable visible shapes.",
+            "stripNamespaces": (
+                "Namespaces will be stripped off of the node before being "
+                "written to Alembic."
+            ),
+            "uvWrite": (
+                "Uv data for PolyMesh and SubD shapes will be written to the "
+                "Alembic file."
+            ),
+            "uvsOnly": (
+                "If this flag is present, only uv data for PolyMesh and SubD "
+                "shapes will be written to the Alembic file."
+            ),
+            "verbose": "Prints the current frame that is being evaluated.",
+            "wholeFrameGeo": (
+                "Data for geometry will only be written out on whole frames."
+            ),
+            "worldSpace": "Any root nodes will be stored in world space.",
+            "writeColorSets": "Write vertex colors with the geometry.",
+            "writeFaceSets": "Write face sets with the geometry.",
+            "writeNormals": "Write normals with the deforming geometry.",
+            "writeUVSets": (
+                "Write all uv sets on MFnMeshes as vector 2 indexed geometry"
+                " parameters with face varying scope."
+            ),
+            "writeVisibility": (
+                "Visibility state will be stored in the Alembic file. "
+                "Otherwise everything written out is treated as visible."
+            )
+        }
+        tooltip = ""
+        for flag in flags:
+            tooltip += "{} - {}\n".format(flag, tooltips[flag])
+        defs.append(
+            EnumDef(
+                "flags",
+                flags,
+                default=enabled_flags,
+                multiselection=True,
+                label="Export Flags",
+                tooltip=tooltip,
+            )
+        )
+
+        for key, value in override_defs.items():
+            if key not in overrides:
+                continue
+
+            defs.append(value)
+
+        defs.append(
+            UISeparatorDef("sep_alembic_options")
```

Comment:
As far as I know the actual keys here _should_ be unique. Right @iLLiCiTiT ? So technically this matching the key/name used above would be invalid?

Reply:
Does not seem to error and the UI looks correct, but will leave to @iLLiCiTiT to confirm on code practice.

Reply:
Yep, multiselection would change the look of it.

Reply:
```suggestion
            UISeparatorDef("sep_alembic_options_end")
```